src/app/app.component.ts

```diff
@@ -7,7 +7,11 @@ import { Component } from '@angular/core';
   selector: 'aou-app',
   styleUrls: [ './app.component.css' ],
   template: `
-    <h1>{{title}}</h1>
+    <div style="display: flex; align-items: flex-end;">
+      <img src="images/all-of-us-logo.svg" alt="All of Us Logo" style="width: 18rem;">
+      <img src="images/portal-logo.svg" alt="Researcher Portal Logo"
+        style="margin-left: 0.5rem; width: 13rem;">
+    </div>
```

Comment:
I don't think IDs buy us anything over class names—I'd still have to invent one for each element to which I want to apply style. The additional consideration with IDs is that they should be globally-unique, so we wouldn't be able to use them on components that are expected to be used more than once on a page.